File: examples/Ta/poddesc.jl
cdir = pwd(); ii = findlast("POD", cdir); PODpath = cdir[1:ii[end]] * "/";    
push!(LOAD_PATH, PODpath * "src");
using POD

# path to LAMMPS executive 
lammps = "/Users/ngoccuongnguyen/lammps/build/lmp";
pod = "fpod_param.pod";
datapath = "training";

# compute POD descriptors for the training dataset 
gd, gdd, na = POD.calculatedescriptors(lammps, pod, datapath);

File: examples/InP/poddesc.py
# import external modules
import numpy, os, sys

# Add POD to Python search path
cdir = os.getcwd(); ii = cdir.find("POD");
srcdir = cdir[0:(ii+3)] + "/"  + "/src";
sys.path.append(srcdir);

# import POD module
import POD

# path to LAMMPS executive 
lammps = "/Users/ngoccuongnguyen/lammps/build/lmp";

# compute POD descriptors for the training dataset 
gd, gdd, na = POD.calculatedescriptors(lammps, "fpod_param.pod", "training");

